.github/workflows/pr-checker.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           format: 'table'
           severity: 'CRITICAL,HIGH'
           vuln-type: 'os,library'
-          skip-dirs: 'opt/telegraf'
+          skip-dirs: 'opt/telegraf,usr/sbin/telegraf'
```

Comment:
Grace, so i am assuming that we are not taking in a newer telegraf as part of this? I remember you said that you tested the newer version, so just checking. I understand the newer version has a few vulnerabilities as well..

Reply:
Right we can't really take a newer version since any newer version has the `monitor_kubernetes_pods=true` feature broken. I have created an [issue](https://github.com/influxdata/telegraf/issues/9600) and am about to make a PR for it. Or we would need to use our own custom build